app/javascript/react/components/molecules/Calendar/CalendarHook.ts

```diff
@@ -105,6 +105,7 @@ const useCalendarHook = ({
   };
 
   useEffect(() => {
+    console.log("On appear, all data considered for calendar: ", movingCalendarData)
```

Comment:
Just to make it shorter I'd change it to `"On appear, all data for calendar: "`
Feel free to ignore this :)